apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -585,6 +585,22 @@ export default function Success(props: PageProps) {
                             </div>
                           </>
                         )}
+                        {rescheduledToUid && (
+                          <>
+                            <div className="mt-3 font-medium">{t("rescheduled_to")}</div>
+                            <div className="col-span-2 mb-2 mt-3">
+                              <span className="underline">
+                                <Link href={`/booking/${rescheduledToUid}`}>
+                                  <div className="flex items-center gap-1">
+                                    {t("Link")}
+                                    <Icon name="external-link" className="h-4 w-4" />
+                                  </div>
+                                </Link>
+                              </span>
+                            </div>
+                          </>
+                        )}
```

Comment:
Let's move it out into it's own component within this module.

Possible name could be <RescheduledToLink>

```
{rescheduledToUid ? <RescheduledToLink>: null}
```